app/bundles/CoreBundle/Helper/CacheHelper.php

```diff
@@ -79,7 +94,7 @@ private function clearConfigOpcache(): void
             return;
         }
 
-        opcache_invalidate($this->configFile, true);
+        opcache_invalidate($this->pathsHelper->getLocalConfigurationFile(), true);
```

Comment:
Just wanted to get this out of the constructor as it might lead to weird issues in case not all paths have been initialized yet